client/scss/components/_dropdown.legacy.scss

```diff
@@ -123,6 +123,29 @@
 
         li {
             border-width: 0 0 1px;
+
+            a:focus,
+            button:focus {
+                border: 3px solid #ffbf47;
+            }
+        }
+
+        @media(forced-colors:$media-forced-colours) {
```

Comment:
to make it easier to find these kinds of changes in the code and also explain 'why', it would be good to change this to 

```
// Media for Windows High Contrast

@media (forced-colors: $media-forced-colours) {
```
So that there is a comment above the change and also a space between the colon & before the bracket.

(1 styles/focus)